### ISSUE TITLE
Update to latest tslint-loader breaks usage with happypack

### ISSUE THREAD

**Issue by Strate** [amireh/happypack#152]
`tslint-loader@3.5.2` calls `emitWarning` with `Error` instance, which is not properly serializes-deserializes when passing from worker to main loader. As a result, it comes to main `emitWarning` as empty object:

```
WARNING in ./src/lib/components/CTab/CTab.tsx
(Emitted value instead of an instance of Error) [object Object]
```

I've put `console.trace` there and this is call trace:
```
at new NonErrorEmittedError (/devel/src/hotfix/megaplan/front/node_modules/webpack/lib/NormalModule.js:51:11)
    at Object.emitWarning (/devel/src/hotfix/megaplan/front/node_modules/webpack/lib/NormalModule.js:115:16)
    at Object.emitWarning (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/HappyRPCHandler.js:79:12)
    at HappyRPCHandler.execute (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/HappyRPCHandler.js:56:22)
    at ChildProcess.acceptMessageFromWorker (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/HappyThread.js:83:27)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at process.nextTick (internal/child_process.js:744:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```
which is comes from happypack as:
```
 at HappyFakeLoaderContext.loader.emitWarning (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/HappyFakeLoaderContext.js:42:13)
    at report (/devel/src/hotfix/megaplan/front/node_modules/tslint-loader/index.js:70:3)
    at lint (/devel/src/hotfix/megaplan/front/node_modules/tslint-loader/index.js:62:3)
    at Object.module.exports (/devel/src/hotfix/megaplan/front/node_modules/tslint-loader/index.js:128:3)
    at applySyncOrAsync (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/applyLoaders.js:341:21)
    at apply (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/applyLoaders.js:268:5)
    at /devel/src/hotfix/megaplan/front/node_modules/happypack/lib/applyLoaders.js:131:7
    at applyPitchLoader (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/applyLoaders.js:184:14)
    at applyPitchLoader (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/applyLoaders.js:192:14)
    at applyPitchLoaders (/devel/src/hotfix/megaplan/front/node_modules/happypack/lib/applyLoaders.js:222:4)
```

**Comment by amireh** [comment]
I believe this should've been fixed in `4.0.0-beta.2` - please re-open if that's not the case.

**Comment by willisplummer** [comment]
(beta fixed for me)
